protocol-designer/src/organisms/AnnouncementModal/index.tsx

```diff
@@ -27,12 +37,18 @@ export const AnnouncementModal = (): JSX.Element => {
   const userHasNotSeenAnnouncement =
     getLocalStorageItem(localStorageAnnouncementKey) !== announcementKey
 
+  const userClickViewReleaseNotes =
+    location.pathname === '/settings' && isViewReleaseNotes === true
+
   const [
     showAnnouncementModal,
     setShowAnnouncementModal,
-  ] = React.useState<boolean>(userHasNotSeenAnnouncement)
+  ] = React.useState<boolean>(
+    userClickViewReleaseNotes || userHasNotSeenAnnouncement
```

Comment:
i think this should be enough logic! 

```suggestion
    userClickViewReleaseNotes || isViewReleaseNotes
```